integration-tests/solr/pom.xml

```diff
@@ -80,4 +98,52 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <!-- force the locale as we want to explicitly test message interpolation -->
+                        <javax.net.ssl.trustStore>${solr.trust-store}</javax.net.ssl.trustStore>
+                        <javax.net.ssl.trustStorePassword>${solr.trust-store-password}</javax.net.ssl.trustStorePassword>
+                    </systemPropertyVariables>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <id>native</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.package.type>native</quarkus.package.type>
+                <quarkus.native.additional-build-args>-H:IncludeResources=.*ssl/.*,-J-Djavax.net.ssl.trustStore=${solr.trust-store},
+                    -J-Djavax.net.ssl.trustStorePassword=${solr.trust-store-password}</quarkus.native.additional-build-args>
```

Comment:
Is there any reason not to put this as `quarkus.native.additional-build-args` to application.properties? Maybe because the local path has to be absolute?

Reply:
yes!! that's the problem. I've actually copied that way of adding these attribute from one of the tests in quarkus project